### PR TITLE
Php8 support and move to GitHub Actions 

### PR DESCRIPTION
**Pull request by tmp-hallenser**
* Using now GitHub Actions instead of travis
* Add imagick for php8 (compile from source)

### CODE REVIEW

**Comment by tmp-hallenser** [comment]
@ildyria Many thanks!!!

File: composer.json [hunk 1]
```diff
@@ -14,14 +14,14 @@
     "require": {
         "php": ">=7.2",
         "neitanod/forceutf8": "^2.0.4",
-        "php-ffmpeg/php-ffmpeg": "^0.14.0"
+        "php-ffmpeg/php-ffmpeg": "dev-master"
```
Comment (ildyria) [inline composer.json:17]: Why `dev-master` and not `^0.17.0` ? 
It seems that https://github.com/PHP-FFMpeg/PHP-FFMpeg/releases/tag/v0.17.0 add support for php8. I fear that "dev-master" would put us at risk to have a broken release at some point.

**Review by ildyria — APPROVED** [review]
Looks good to me aside from the dependency version on PHP-FFMpeg/PHP-FFMpeg

**Comment by ildyria** [comment]
:+1:

**Pull request merged**